src/crimson/os/seastore/segment_cleaner.h

```diff
@@ -751,11 +837,27 @@ class SegmentCleaner : public SegmentProvider {
   using work_iertr = ExtentCallbackInterface::extent_mapping_iertr;
 
 private:
+  double calc_utilization(segment_id_t id) const {
+    auto& info = segments[id];
+    if (info.is_open()) {
+      // bucket 11 stores open segments
+      return 1.05;
```

Comment:
See above, I think this would be a bit too tough for a reader to follow.

Reply:
Fixed in https://github.com/ceph/ceph/pull/46158/commits/eb9ffca28292e5a5f45706953d9f7e016f7591ff